nucliadb/nucliadb/ingest/serialize.py

```diff
@@ -71,11 +71,18 @@ async def set_resource_field_extracted_data(
         if data_et is not None:
             field_data.text = models.ExtractedText.from_message(data_et)
 
-    if ExtractedDataTypeName.METADATA in wanted_extracted_data:
+    metadata_wanted = ExtractedDataTypeName.METADATA in wanted_extracted_data
+    small_metadata_wanted = (
+        ExtractedDataTypeName.SMALL_METADATA in wanted_extracted_data
+    )
+    if metadata_wanted or small_metadata_wanted:
         data_fcm = await field.get_field_metadata()
 
         if data_fcm is not None:
-            field_data.metadata = models.FieldComputedMetadata.from_message(data_fcm)
+            small_version = small_metadata_wanted and not metadata_wanted
```

Comment:
Large metadata is another key in the maindb.
The problem we are trying to address here is that the contents of METADATA can be too big sometimes and the dashboard wants to avoid sending some of it's inner attributes over the network